src/components/AdminSettings.vue

```diff
@@ -8,103 +8,98 @@
-						</span>
-						<ul>
-							<li><a href="https://github.com/nextcloud/llm2#readme">Local Large language model app</a></li>
-							<li><a href="https://apps.nextcloud.com/apps/integration_openai">OpenAI/LocalAI Integration</a></li>
-						</ul>
-					</div>
-				</NcNoteCard>
-				<NcCheckboxRadioSwitch
-					:model-value="state.text_to_image_picker_enabled"
-					:disabled="!state.text_to_image_picker_available"
-					@update:model-value="onCheckboxChanged($event, 'text_to_image_picker_enabled')">
-					<div class="checkbox-text">
-						{{ t('assistant', 'Enable text-to-image in smart picker') }}
-					</div>
-				</NcCheckboxRadioSwitch>
-				<NcCheckboxRadioSwitch
-					:model-value="state.text_to_sticker_picker_enabled"
-					:disabled="!state.text_to_image_picker_available"
-					@update:model-value="onCheckboxChanged($event, 'text_to_sticker_picker_enabled')">
-					<div class="checkbox-text">
-						{{ t('assistant', 'Enable text-to-sticker in smart picker') }}
-					</div>
-				</NcCheckboxRadioSwitch>
-				<NcNoteCard v-if="!state.text_to_image_picker_available" type="warning">
-					<div class="checkbox-text">
-						<span>
-							{{ t('assistant', 'To enable the sticker generation picker or the image generation picker, please install and enable a "Generate image" provider:') }}
-						</span>
-						<ul>
-							<li><a href="https://github.com/nextcloud/text2image_stablediffusion#readme">Local Text-To-Image StableDiffusion</a></li>
-							<li><a href="https://apps.nextcloud.com/apps/integration_openai">OpenAI/LocalAI Integration</a></li>
-						</ul>
-					</div>
-				</NcNoteCard>
-				<NcCheckboxRadioSwitch
-					:model-value="state.speech_to_text_picker_enabled"
-					:disabled="!state.speech_to_text_picker_available"
-					@update:model-value="onCheckboxChanged($event, 'speech_to_text_picker_enabled')">
-					<div class="checkbox-text">
-						{{ t('assistant', 'Enable speech-to-text in smart picker') }}
-					</div>
-				</NcCheckboxRadioSwitch>
-				<NcNoteCard v-if="!state.speech_to_text_picker_available" type="warning">
-					<div class="checkbox-text">
-						<span>
-							{{ t('assistant', 'To enable speech-to-text in the smart picker, please install and enable a "Transcribe audio" provider:') }}
-						</span>
-						<ul>
-							<li><a href="https://github.com/nextcloud/stt_whisper2#readme">Local Speech-To-Text Whisper</a></li>
-							<li><a href="https://apps.nextcloud.com/apps/integration_openai">OpenAI/LocalAI Integration</a></li>
-						</ul>
-					</div>
-				</NcNoteCard>
+				<NcFormGroup :label="t('assistant', 'Select which features you want to enable')"
+					class="switch-group">
+					<NcFormBox>
+						<NcFormBoxSwitch :model-value="state.assistant_enabled"
+							@update:model-value="onCheckboxChanged($event, 'assistant_enabled')">
+							{{ t('assistant', 'Enable Nextcloud Assistant in header') }}
```

Comment:
```suggestion
							{{ t('assistant', 'Nextcloud Assistant in header') }}
```

A good time to get rid of all the "Enable"s in these options. They are redundant.